src/visualization/actions/index.ts

```diff
@@ -9,7 +9,9 @@ export const enum Types {
     ToggleFullPage,
     NotFound,
     ToggleInactiveSession,
-    Error
+    Error,
+    ShowSettleEvents,
```

Comment:
>ShowSettleEvents [](start = 4, length = 16)

as discussed offline, let's move away from calling this settle events or settle time. It would be better branded as perceivedLoadTime. 

For this particular Types definition, maybe ShowPltVizualization ?

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175850109](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175850109) [](ancestors = 175850109)